docs/proposals/project-rolebinding-restrictions.md

```diff
@@ -0,0 +1,58 @@
+# Project RoleBinding Restrictions
+
+## Problem
+A project-admin may not be allowed to bind any subject he chooses into his project.  Doing so can produce problems
+for the bindee like unexpected typo shadowing or an unviewable console (too many projects).  On the binder side,
+the project-admin may accidentally expose confidential projects to unauthorized users (think private repos on github).
+
+To prevent those issues, a cluster-admin should be able to restrict the set of subjects (users, groups, serviceaccounts)
```

Comment:
So there's the cluster admin aspect of this, but the more important aspect is the end user aspect.  I think ultimately this is an end user problem.  Does this proposal address how end users can resolve this, or is this focused only on system admin?

I'd kind of prefer to not fix this for cluster admins, but fix it for end users.